### PR TITLE
Add cmplxfoil recipe

### PR DESCRIPTION
**Pull request by whophil**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by whophil** [comment]
@bernardopacini @eytanadler @A-CGray please confirm if you want to be listed as maintainers of this recipe.

Also, note this requirement from conda-forge.
> Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).

AFAIK, `cmplxfoil` vendors (a modified version of) XFOIL code, but does not bundle the original XFOIL license with it. This should probably be fixed upstream (in the `cmplxfoil` project).

File: recipes/cmplxfoil/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+about:
+  home: https://github.com/mdolab/CMPLXFOIL/
+  license: GPL-2.0-or-later
+  license_family: GPL
+  license_file: LICENSE.md
+  summary: CMPLXFOIL is a version of Mark Drela's XFOIL code with the GUI features removed.
+  description: |
+     CMPLXFOIL is a version of Mark Drela's XFOIL code with the GUI features removed. 
+     radient computation is implemented with the complex-step method.
```
Comment (whophil) [inline recipes/cmplxfoil/meta.yaml:46]:
```suggestion
     Gradient computation is implemented with the complex-step method.
```

**Comment by A-CGray** [comment]
> @bernardopacini @eytanadler @A-CGray please confirm if you want to be listed as maintainers of this recipe.
> 

We're happy to be listed as maintainers.

> Also, note this requirement from conda-forge.
> 
> > Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
> 
> AFAIK, `cmplxfoil` vendors (a modified version of) XFOIL code, but does not bundle the original XFOIL license with it. This should probably be fixed upstream (in the `cmplxfoil` project).

CMLXFOIL uses the same GPL license as the original XFOIL, so I think the license in the top level of the repo covers both?

**Comment by whophil** [comment]
@A-CGray sounds good, the conda-forge reviewers can probably let us know if this passes muster.

I believe @bernadopacini and @eytanadler will need to individually comment on this PR to confirm they are OK with being maintainers.

**Comment by bernardopacini** [comment]
Thank you for doing this @whophil! I am good with being listed as a maintainer too.

**Comment by eytanadler** [comment]
Sorry for the delay, I can be a maintainer too. Thanks again @whophil!

**Comment by whophil** [comment]
@conda-forge/help-python this is ready for review!

**Comment by whophil** [comment]
Thanks @wolfv !

**Pull request merged**